README.md

```diff
@@ -1,10 +1,5 @@
 # Amazon EKS Blueprints for Terraform
 
-[![e2e-test](https://github.com/aws-ia/terraform-aws-eks-blueprints/actions/workflows/e2e-test.yml/badge.svg)](https://github.com/aws-ia/terraform-aws-eks-blueprints/actions/workflows/e2e-test.yml)
```

Comment:
can we update these to use the current workflows?

Reply:
We can, yes. However, you typically have badges for checks that show the current status of the project and we don't have any workflows that run post-merge to `main`. So do we want to make one or more of the workflows run when commits land on `main` and which workflow(s) would that be?

Reply:
Updated in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/503/commits/73261067bf46572802cce2086a8a220fea275b8c